Client/game_sa/CFxSA.cpp

```diff
@@ -237,3 +241,42 @@ void CFxSA::TriggerFootSplash(CVector& vecPosition)
         call    dwFunc
     }
 }
+
+bool CFxSA::IsShadowsLimitReached()
+{
+    short shadows = *(short*)0xC403DC;

```

Comment:
does this access some global in memory? if the struct is already defined, you should use it, and if necessary extend. cc @saml1er 

(if saml1er does not respond, happy to overlook)